### PR TITLE
[10.0][MIG] web_search_autocomplete_prefetch

### PR DESCRIPTION
**Pull request by hbrunn**
relevant commit is https://github.com/OCA/web/commit/cc0a2c5dbf9f508a06f413c19182816c8dae9437

### CODE REVIEW

File: web_search_autocomplete_prefetch/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Prefetch autocomplete offers",
+    "version": "10.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Usability",
```
Comment (JayVora-SerpentCS) [inline web_search_autocomplete_prefetch/__manifest__.py:10]: Website : https://github.com/OCA/web

File: web_search_autocomplete_prefetch/README.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+When searching, the autocomplete options can be a bit frustrating because you
+will be offered choices that won't yield a result. This addon searches for the
+term in the background and only offers an option if this search has a result.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot/162/8.0
```
Comment (JayVora-SerpentCS) [inline web_search_autocomplete_prefetch/README.rst]: 10.0 plz

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

File: web_search_autocomplete_prefetch/README.rst (hunk 2 above)
Comment (hbrunn) [reply web_search_autocomplete_prefetch/README.rst]: the readme is generated by now, no use doing stuff there manually

File: web_search_autocomplete_prefetch/views/base_view.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_view_search_autocomplete">
+        <field name="model">ir.ui.view</field>
+        <field name="inherit_id" ref="base.view_view_search"/>
+        <field name="arch" type="xml">
+            <field name="name" position="attributes">
+                <attribute name="options">{'web_search_autocomplete_prefetch.disable': true}</attribute>
```
Comment (thomaspaulb) [inline web_search_autocomplete_prefetch/views/base_view.xml]: Shouldn't this have a `t-translation="off"` ? A lot of the translations show up this value.

File: web_search_autocomplete_prefetch/i18n/fi.po [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+"Language: fi\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: web_search_autocomplete_prefetch
+#: view:ir.ui.view:web_search_autocomplete_prefetch.view_view_search_autocomplete
+msgid "{'web_search_autocomplete_prefetch.disable': true}"
+msgstr "{'web_search_autocomplete_prefetch.disable': true}"
```
Comment (thomaspaulb) [inline web_search_autocomplete_prefetch/i18n/fi.po:24]: None of these translation files contain anything useful.

File: web_search_autocomplete_prefetch/README.rst (hunk 2 above)
Comment (thomaspaulb) [reply web_search_autocomplete_prefetch/README.rst]: @JayVora-SerpentCS any other comments on this module?

File: web_search_autocomplete_prefetch/i18n/fi.po (hunk 4 above)
Comment (hbrunn) [reply web_search_autocomplete_prefetch/i18n/fi.po:24]: those we can ignore, they will be regenerated by the weblate bot

**Comment by hbrunn** [comment]
@JayVora-SerpentCS @thomaspaulb thanks for the review, attended to your points

**Review by JayVora-SerpentCS — APPROVED** [review]
@hbrunn Perfect as always.

**Comment by pedrobaeza** [comment]
Is travis fail related?

**Comment by hbrunn** [comment]
nope, that's the [flimsy phantomjs tests](https://travis-ci.org/OCA/web/jobs/436723636#L1572-L1586) of `web_widget_slickroom` which make all of the web branches red for a long time already. I'm of a mind sometimes to just PR to remove the module until someone fixes the tests, but as we don't have a procedure for that, never followed through.

I do however have `base_test_chrome` or `base_test_firefox` on my modules-to-write-list that enables us to use some headless browser for tests, that should be much more stable, and also not restrict us to ancient JS because of phantomjs. Odoo v12 does something very similar.

File: web_search_autocomplete_prefetch/__init__.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2018 Therp BV <http://therp.nl>
```
Comment (nikul-serpentcs) [inline web_search_autocomplete_prefetch/__init__.py]: @hbrunn Remove Copyright from all `__init__` file.

**Review by nikul-serpentcs — COMMENTED** [review]
Small Change

**Comment by pedrobaeza** [comment]
@hbrunn maybe we can simply remove the test (or the trigger of the test).

**Comment by hbrunn** [comment]
usually that's a horrible way to `fix` tests, but given I don't have time to dive into fixing this properly (there's some [js syntax error](https://travis-ci.org/OCA/web/jobs/436723636#L1414) from the point of view of phantomjs, could be just a misplaced comma), that's fine with me

**Comment by pedrobaeza** [comment]
I know, but it's a module that is not mainstream... May you?

**Comment by hbrunn** [comment]
may I? don't understand

**Comment by hbrunn** [comment]
but green, yay!

**Comment by hbrunn** [comment]
will be green once #1073 is merged

**Review by simahawk — APPROVED** [review]
relevant commit looks good. 
#1073 is merged.
I triggered re-build just to make sure recent changes did not trick us after merge.

**Comment by pedrobaeza** [comment]
It seems there's still some comment unattended, isn't it?

**Comment by ovnicraft** [comment]
@pedrobaeza all important comments was solved, must merged IMO

**Pull request merged**